modules/k6/k6.go

```diff
@@ -20,14 +20,17 @@ type K6Container struct {
 func WithTestScript(scriptPath string) testcontainers.CustomizeRequestOption {
 	return func(req *testcontainers.GenericContainerRequest) {
 		script := filepath.Base(scriptPath)
-		target := fmt.Sprintf("/tests/%s", script)
-		mount := testcontainers.ContainerMount{
-			Source: testcontainers.GenericBindMountSource{
-				HostPath: scriptPath,
+		target := filepath.Join("/home/k6x", script)
```

Comment:
I guess this is the expected path from where k6 will read the test scripts, right?

```diff
- filepath.Join("/home/k6x", script)
+ "/home/k6x/" + script
```

As we know there is a Linux inside that container